.github/workflows/lint.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: "3.8"
```

Comment:
python version 3.8 would raise a warning related to setup tool. Switching to python 3.9 somehow solves it.